src/main/resources/static/js/MainWindow.js

```diff
@@ -312,19 +312,19 @@ function saveConfig() {
 	} else if ($("input[name=source]:eq(1)").is(":checked")) {
 		filetype = $("#sourceinput").val();
 	}
-	if (!(filetype) || filetype === "") {
+	if (!(filetype) || filetype == "") {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)